RevenueCatUI/Templates/V2/Previews/PreviewMock.swift

```diff
@@ -78,7 +78,6 @@ struct PreviewRequiredEnvironmentProperties: ViewModifier {
     let screenCondition: ScreenCondition
     let componentViewState: ComponentViewState
     let packageContext: PackageContext?
-    let colorScheme: ColorScheme
```

Comment:
`colorScheme` is always present in the environment as it's a default SwiftUI property. Hardcoding it in `previewRequiredEnvironmentProperties` was making it impossible to switch the color scheme in Xcode previews.